frontend/candolim-beach/index.html

```
@@ -0,0 +1,389 @@
+                </div>
+
+            </div>
+        </section>
+
+    </main>
+
+    <!-- Lightbox Modal for Gallery Images -->
+    <div class="lightbox-modal" id="lightbox-modal" role="dialog" aria-modal="true" aria-label="Expanded image viewer with attribution">
+        <div class="lightbox-content">
+            <button class="lightbox-close" id="lightbox-close-btn" aria-label="Close image modal">&times;</button>
+            <img src="" id="lightbox-img" alt="">
```

HTMLHint (1.9.2)

[error] The attribute [ src ] of the tag [ img ] must have a value.

(src-not-empty)